install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -81,6 +81,24 @@ spec:
               description: provisioningOSDownloadURL is the location from which the OS Image used to boot
                 baremetal host machines can be downloaded by the metal3 cluster.
               type: string
+            provisioningNetwork:
+              description: provisioningNetwork provides a way to indicate the state of the underlying
+                network configuration for the provisioning network. This field can have one of the
+                following values -
+                `managed`- when the provisioning network is completely managed by the Baremetal IPI
+                solution.
+                `unmanaged`- when the provsioning network is present and used but the user is
+                responsible for managing DHCP. Virtual media provisioning is recommended but PXE
+                is still available if required.
+                `disabled`- when the provisioning network is fully disabled. User can bring up the
+                baremetal cluster using virtual media or assisted installation. If using metal3 for
+                power management, BMCs must be accessible from the machine networks. User should
+                provide two IPs on the external network that would be used for provisioning services.
```

Comment:
Your help message suggests the three options are `managed`, `unmanaged` and `disabled`, however the enum list has these with capitals at the beginning, might be good to update the description to match the casing of the accepted values in the enum

Reply:
Done.